src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -13,6 +13,8 @@
 class ThetaStarPathPlanner : public PathPlanner
 {
    public:
+    ThetaStarPathPlanner();
```

Comment:
Is this just for testing?

Reply:
Ah, this is because we should be giving class members explicit default values, rather then just relying on the default constructors.